x-pack/solutions/search/plugins/enterprise_search/public/applications/enterprise_search_content/components/connector_detail/attach_index_box.tsx

```diff
@@ -99,7 +99,7 @@ export const AttachIndexBox: React.FC<AttachIndexBoxProps> = ({ connector }) =>
   const [sanitizedName, setSanitizedName] = useState<string>(
     prefixConnectorIndex(formatApiName(connector.name))
```

Comment:
Since we are defining this we can use this in here as well
```suggestion
    prefixConnectorIndex(sanitizedNameWithoutPrefix)
```